src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -544,8 +542,6 @@ private static int preallocatedHashmapCapacity(int elementsToHold) {
         final Closure body;
         final List<String> msgs;
         final Set<String> interpolatedStrings;
-        // UninstantiatedDescribable is set when the associated symbol is being built as the parameter of a step to be invoked.
-        UninstantiatedDescribable uninstantiatedDescribable = null;
 
         private NamedArgsAndClosure(Map<?,?> namedArgs, Closure body, @Nonnull Set<String> foundInterpolatedStrings) {
```

Comment:
@car-roll Do we need to do any cleanup in `NamedArgsAndClosure` now? I guess we should remove `implements Serializable` and remove "The instance is returned to the Groovy program ..." from the Javadoc.

Reply:
Technically the whole paragraph might have to go, right? Because we are now using the `UninstantiatedDescribable` again. Maybe that paragraph should go to `UninstantiatedDescribableWithInterpolation`?

Reply:
Yeah you are right, everything other than "This class holds the argument map and optional body of the step that is to be invoked." is no longer accurate. Feel free to move it to the new class if you want.